data/campaigns/Heir_To_The_Throne/scenarios/23_Northern_Winter.cfg

```diff
@@ -0,0 +1,693 @@
+                message_Ulfdain=_"We ain’t prepared t’ fight ferever in a blizzard like this, Konrad. Lest ye’ve got some very fast warriors, we may be havin’ to split up t’ get all o’ Jeniver’s ingredients afore we run out o’ time and the cold drives us away. Them northeastern mountains look ’specially hard t’ reach!"
+                message_Dosh=_"Storm too cold, Konrad. Unless you got some fast fighters, Dosh is thinking we’ve got no time get all da ’gredients unless you splits us up. Otherwise we get too cold and den have to leave. Northeast mountains look ’specially hard to be gettin’ to."
+                fallback_Konrad=_"We won’t be able to fight in this blizzard forever. Unless we have very fast warriors, we may have to split up if we want to collect all of your ingredients before we run out of time and the cold drives us away. The northeastern mountains look especially hard to reach!"
+            [/companion_message]
+            [message]
+                speaker=Jeniver
+                message=_"True, true, but, well, you know, I suppose there’d really be no harm in coming with you even if you don’t get them all— yes, I think I’ll join you no matter what. You seem the nice sort, very chivalrous and all that."
+            [/message]
+            [message]
+                speaker=Jeniver
+                message=_"But— err, of course I suppose I wouldn’t really be of much use to you unless you can gather at least some ingredients so that I can get my — you know — alchemical concoctions prepared. The more the better!"
+            [/message]
+        [/event]
+        # Jeniver's slow would normally end on side 2 turn end. But that makes it look like she's not getting slowed at all, so sync her up to side 1 instead
+        [event]
+            name=side 2 turn
+            first_time_only=no
+            {FILTER_CONDITION({HAVE_UNIT id,status=Jeniver,slowed})}
+            [event]
+                name=side 3 turn
+                {MODIFY_UNIT id=Jeniver status.slowed yes}
+            [/event]
+            [event]
+                name=side 1 turn end
+                {MODIFY_UNIT id=Jeniver status.slowed no}
+            [/event]
+        [/event]
+
+        #############################
+        # SMALLTALK
+        #############################
+        [event]
+            name=turn 5
+            {FIRE_EVENT say_smalltalk}
+        [/event]
+        [event]
+            name=turn 15
+            {FIRE_EVENT say_smalltalk}
+        [/event]
+
+        #############################
+        # EXPLAIN CAVES
+        #############################
+        [event]
+            name=moveto
+            {FILTER side,x,y=1,32-99,23-99}
+            {IF} {VARIABLE_CONDITIONAL s23_from equals e} {THEN(
+                [message]
+                    speaker=Konrad
+                    message=_"These are the Knalgan caves exited out of. No point going back, at least not until we’ve finished helping Jeniver collect her ingredients."
```

Comment:
The first sentence doesn't make sense to me - is there a word or subject missing?